shell/app/modules/project/pages/backlog/backlog.scss

```diff
@@ -22,6 +22,10 @@
         height: calc(100% - 40px);
         overflow: auto;
       }
+
+      .ant-pagination-item-link {
```

Comment:
one question. Whether need move this to antd-extension ?  is there any other place may have the same problem?

Reply:
leave a comment or push relate commit before resolve the conversation